lib/core/src/Cardano/Wallet/Primitive/Fee.hs

```diff
@@ -93,6 +94,24 @@ computeFee policy (Quantity sz) =
   where
     LinearFee (Quantity a) (Quantity b) (Quantity _c) = policy
 
+-- | Compute fee for a delegation certificate. One need to take into
+-- consideration that "regular tx" fee calculation still holds. There
+-- is additional flat fee inscribed on top of that:
+--
+-- @
+--     f = a + size * b + c
+-- @
+--
+-- where @a@ & @b@ are values fixed by the protocol.
+computeCertFee
+    :: FeePolicy
+    -> Quantity "byte" Int
+    -> Fee
+computeCertFee policy (Quantity sz) =
+    Fee $ ceiling (a + b*fromIntegral sz + c)
```

Comment:
I am here assuming that this is flat fee for certificates and I can delegate to one stake pool. If this assumption is not true, then I should change the signature of this function and add `number of certificates`. On the other hand I see, something like this : https://slack-redir.net/link?url=https%3A%2F%2Fgithub.com%2Finput-output-hk%2Fchain-libs%2Fblob%2Fmaster%2Fchain-impl-mockchain%2Fdoc%2Fformat.md%23type-0-initial-blockchain-configuration and here 
> 
LinearFee has the format:

Constant | Coefficient | Certificate
all of them 64-bit unsigned integers, specifying how fees are computed using the formula:

Constant + Coefficient * (inputs + outputs) + Certificate * certificates
where inputs, outputs and certificates represent the size of the serialization of the corresponding parts of a transaction in bytes.
> 
so according to this document `certificates` is size of serialization part

Reply:
I agree that this function should probably take an extra argument to reflect the number of certificates. And, at the call-site, use a value of `1` since this is where we actually _know_ that we are only creating a single certificate.